### PR TITLE
removed fonttools formula, which is now in homebrew-core

### LINKED ISSUE #16: fonttools is in homebrew-core
https://github.com/Homebrew/homebrew-core/blob/master/Formula/fonttools.rb

### PR DESCRIPTION
**Pull request by verhovsky**
This is how I would close https://github.com/bramstein/homebrew-webfonttools/issues/16

Right now if you do `brew tap bramstein/webfonttools; brew install fonttools` it will install the one in `homebrew-core` anyway, so no one should notice that this formula was delete, deprecation is overkill (famous last words).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,16 @@
...
 
-Add the tap:
+    brew tap bramstein/webfonttools
+    brew install woff2
 
-    > brew tap bramstein/webfonttools
-
-Update brew:
-
-    > brew update
```
Comment (verhovsky) [inline README.md:9]: Brew `update`s on every `install` by default.

File: README.md [hunk 2]
```diff
@@ -1,21 +1,16 @@
 # Homebrew web font tools
 
-Add the tap:
+    brew tap bramstein/webfonttools
```
Comment (BorisVerk) [inline README.md]: prefixing with `>` makes copy paste harder

File: README.md [hunk 3]
```diff
@@ -1,21 +1,16 @@
...
+    brew install woff2
 
-    > brew tap bramstein/webfonttools
```
Comment (verhovsky) [inline README.md:5]: prefixing with `>` makes copy paste harder

File: README.md [hunk 4]
```diff
@@ -1,21 +1,16 @@
...
+Or install any of the following formulas with `brew install <formula>`:
 
 * `sfnt2woff`: Tool for converting TrueType and OpenType font files to WOFF made by Jonathan Kew. This installs two utilities `sfnt2woff` to compress and `woff2sfnt` to decompress.
 * `sfnt2woff-zopfli`: Modified version of sfnt2woff that uses the Zopfli compression algorithm (2-5% better compression than standard WOFF). This installs two utilities `sfnt2woff-zopfli` to compress and `woff2sfnt-zopfli` to decompress.
 * `woff2`: WOFF2 compression and decompression utilities by Google. This installs two utilities `woff2_compress` to compress and `woff2_decompress` to decompress.
 * `ttf2eot`: Tool for converting TrueType font files to the embedded OpenType format (EOT). Does not include MTX compression. The installed utility is called `ttf2eot`.
 * `sfntly`: Google's subsetting and font conversion tools. This will install two utilities called `sfnttool` and `fontinfo`. The `sfnttool` utility performs subsetting and can also create WOFF and EOT files. The `fontinfo` utility shows you information about a font.
-* `fonttools`: [TTX/fonttools (Behdad's fork)](https://github.com/behdad/fonttools).
```
Comment (BorisVerk) [inline README.md]: https://github.com/behdad/fonttools redirects to https://github.com/fonttools/fonttools

File: README.md [hunk 5]
```diff
@@ -1,21 +1,16 @@
...
+* `fonttools`: [TTX/fonttools](https://github.com/fonttools/fonttools) (this package is now in [homebrew-core](https://github.com/Homebrew/homebrew-core/blob/master/Formula/fonttools.rb), you can `brew install` it without this tap).
```
Comment (verhovsky) [inline README.md:13]: https://github.com/behdad/fonttools redirects to https://github.com/fonttools/fonttools

People that are searching for web font tools might want to know about `fonttools`.

**Comment by BorisVerk** [comment]
Right now if you do `brew tap bramstein/webfonttools; brew install fonttools` it will install the one in `core` anyway, so no one should notice the difference anyway, deprecation is overkill (famous last words).

**Comment by bramstein** [comment]
This is great. Thanks a lot for making the changes to the README.

**Pull request merged**